QueueManagerTest/DirectoryReaderTests.cs

```diff
@@ -32,15 +61,32 @@ public void TestAddFilesFromNullDirectory()
         [Test]
         public void TestCreateDictionaryForNamesAndPaths()
         {
-            Assert.AreEqual(new Dictionary<string, string> { { path + "a1.iso", "a1" }, { path + "a2.iso", "a2" }, { path + "a3.iso", "a3" }, { path + "a4.iso - Shortcut.lnk", "a4" } },
-                _directoryReader.CreateDictionaryForNamesAndPaths(_directoryReader.GetListOfMoviesFromDirectory(path)));
+            Dictionary<string, string> testDict = new Dictionary<string, string>();
+            foreach(KeyValuePair<string,string> item in _testFiles)
+            {
+                testDict.Add(_upperLevelFolderName + item.Key + item.Value, item.Key);
+            }
+            Assert.AreEqual(testDict, _directoryReader.CreateDictionaryForNamesAndPaths(_directoryReader.GetListOfMoviesFromDirectory(_upperLevelFolderName)));
         }
 
         [Test]
         public void TestGetListOfFileNames()
         {
-            Dictionary<string, string> dictionaryOfPathsAndNames = _directoryReader.CreateDictionaryForNamesAndPaths(_directoryReader.GetListOfMoviesFromDirectory(path));
-            Assert.AreEqual(new List<string> { { "a1" }, { "a2" }, { "a3" }, { "a4" } }, _directoryReader.GetListOfFileNames(dictionaryOfPathsAndNames));
+            List<string> testList = new List<string>();
+            foreach(KeyValuePair<string,string> item in _testFiles)
+            {
+                testList.Add(item.Key);
+            }
+            Dictionary<string, string> dictionaryOfPathsAndNames = _directoryReader.CreateDictionaryForNamesAndPaths(_directoryReader.GetListOfMoviesFromDirectory(_upperLevelFolderName));
+            Assert.AreEqual(testList, _directoryReader.GetListOfFileNames(dictionaryOfPathsAndNames));
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            GC.Collect();
```

Comment:
I see these three lines are repeated in 3 different places. Could these 3 calls be consolidated?